### PR TITLE
Return Proper Appointment Information

### PR DESCRIPTION
**Pull request by CodyBrewer**
## Description

Appointment routes do not return relevant information and would require the frontend to make multiple calls to get information for the  pet name, image_url, service information, location information etc.

This fix returns the appointment in an object as so: 
```json
  {
      "id": 0,
      "groomer_id": "00ultwew80Onb2vOT4x6",
      "customer_id": "00ulthapbErVUwVJy4x6",
      "appointment_date_time": 1611878560,
      "status": "Done",
      "duration": 60,
      "pet": {
        "name": "Rabby",
        "type": "Dog",
        "color": "Red",
        "date_of_birth": "2020-11-02T07:00:00.000Z",
        "image_url": "https://s3.amazonaws.com/1.jpg",
        "health_issue": "Diabetes"
      },
      "service": {
        "service_description": "a service for animals",
        "service_image": null,
        "service_cost": 100
      },
      "location": {
        "location_name": "Buddy",
        "street_address": "200 E Colfax Ave",
        "state": "Colorado",
        "city": "Denver",
        "zip": 80203,
        "country": "United States of America",
        "phone_number": "3038675309",
        "latitude": 39.739445,
        "longitude": -104.985
      }
  }
```

This PR also fixes the appointmentRouter not being able to run on other routes such as profile/:profileId/appointments or customerPets/appointments or customerPets/:customerPetID/appointments. The change to routing was done in the api/app.js file.

Type
- [x] This is a bugfix.
- [ ] This is a new feature.
- [ ] This is part of a feature.
## Organization
- [x] Have you linked this pull request to a Trello card?
- [x] Have you named the branch in the format of feature/describe_feature_or_change OR bug/describe_bug_this_is_fixing
## Review
- [ ] Is this a work in progress?
- [x]  Is this ready to be reviewed?
- [x]  Have you added two reviewers to this pull request?

### CODE REVIEW

**Comment by CodyBrewer** [comment]
[RETURN PET, SERVICE, APPOINTMENT LOCATION INFO AND ALL THAT OTHER STUFF](https://trello.com/c/UpiKNeMm/88-return-pet-service-appointment-location-info-and-all-that-other-stuff)

File: api/app.js [hunk 1]
```diff
@@ -63,7 +63,16 @@ app.use(['/location', '/locations'], locationRouter);
...
 app.use(['/customerPet', '/customerPets'], customerPetRouter);
 app.use('/groomerSearch', groomerSearchRouter);
-app.use(['appointment', '/appointments'], appointmentsRouter);
+app.use(
+  [
+    '/appointment',
+    '/appointments',
+    '*/appointments',
+    '*/appointment',
+    '*/appointments',
```
Comment (ClydeFrog04) [inline api/app.js]: */appointments is here twice, once we get this fixed it should be ready to merge!

**Review by ClydeFrog04 — CHANGES_REQUESTED** [review]
App.js has app.use('*/appointments') twice

**Review by ClydeFrog04 — APPROVED** [review]

**Review by cdlafortune — APPROVED** [review]
Thanks for fixing this!

**Pull request merged**